### PR TITLE
Add select field to folders find method

### PR DESCRIPTION
**Pull request by SubashPradhan**
# Description
- Add `select` field to `GET folders/id` 
- https://github.com/nylas/nylas-ruby/issues/499

# License
I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: lib/nylas/resources/folders.rb [hunk 1]
```diff
@@ -28,9 +28,10 @@ def list(identifier:, query_params: nil)
     # @param identifier [String] Grant ID or email account to query.
     # @param folder_id [String] The id of the folder to return.
```
Comment (AaronDDM) [inline lib/nylas/resources/folders.rb:29]: Ulster the doc comments

File: spec/nylas/resources/folders_spec.rb [hunk 2]
```diff
@@ -42,7 +42,7 @@
       folder_id = "5d3qmne77v32r8l4phyuksl2x"
       path = "#{api_uri}/v3/grants/#{identifier}/folders/#{folder_id}"
       allow(folders).to receive(:get)
-        .with(path: path)
+        .with(path: path, query_params: nil)
```
Comment (AaronDDM) [inline spec/nylas/resources/folders_spec.rb:45]: Add a new test that tests the query param as well

**Comment by bmulholland** [comment]
Thanks for this! Just noticed that threads.find is also missing the same. Any chance we could get that while you're at it?

**Comment by SubashPradhan** [comment]
> Thanks for this! Just noticed that threads.find is also missing the same. Any chance we could get that while you're at it?

Sure, thanks for pointing that out.

**Review by AaronDDM — APPROVED** [review]

**Comment by bmulholland** [comment]
FYI I've tried this out in production and it's working well. Thank you!

**Pull request merged**